src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -671,6 +671,10 @@ public function load(array $configs, ContainerBuilder $container): void
             $this->registerHtmlSanitizerConfiguration($config['html_sanitizer'], $container, $loader);
         }
 
+        if (!$config['use_sapi_error_renderer']) {
```

Comment:
The deprecation message can be triggered here:


```suggestion
        if (! ($useSapiErrorRenderer = $config['use_sapi_error_renderer'] ?? null)) {
            if ($useSapiErrorRenderer === null) {
                trigger_deprecation('symfony/framework-bundle', '7.4', 'Not setting the "framework.use_sapi_error_renderer" config option is deprecated. It will default to "true" in 8.0.');
            }

```

Reply:
I moved the deprecation to the FWB extension as suggested, thanks!